R/sampler-mh.R

```diff
@@ -0,0 +1,61 @@
+##' Create a simple Metropolis-Hastings sampler (really just a plain
+##' Metropolis sampler since we assume symmetric proposals and don't
+##' yet support the correction for asymmetric proposal).
+##'
```

Comment:
I would call this one Random Walk rather than Metropolis-Hastings. MH refers to the acceptance based on the ratio of densities (only model densities for M and model corrected by proposal densities for MH). Metropolis-adjusted Langevin algorithm that can be seen as a proto-algorithm of HMC is a MH algorithm with a slightly more complicated proposal based on the gradient. Has we are using a local multivariate normal proposal here it would be better to refer it as random walk - i.e. a sampler based on "blind" local steps.